apps/webClient/src/components/View/index.tsx

```diff
@@ -99,12 +108,35 @@ const View: React.FC<Props> = ({ children }) => {
         <Box mt={deriveLogoBoxMarginTop()} mb={12}>
           <Grid container item justify="center">
             <Link to="/">
-              <Logo height={deriveLogoHeight()} />
+              <Logo height={deriveLogoHeight()} svgColor="white" />
             </Link>
           </Grid>
         </Box>
         {children}
       </Container>
+      <Box className={classes.footer}>
+        <Box mt={1} mb={1}>
```

Comment:
Too many unnecessary elements here.  There is a lot we can do with box: https://material-ui.com/system/borders/#api
1) use 1 box
 - uses borderTop prop
 - uses padding instead of margin (you can also just use `spacing` on the grid maybe)